workspace/assets/css/core/fct-animation.less

```diff
@@ -12,4 +12,11 @@
 	@-ms-keyframes     @name {@rules();}
 	@-o-keyframes      @name {@rules();}
 	@keyframes         @name {@rules();}
-}
\ No newline at end of file
+}
+#animation-delay (@delay) {
```

Comment:
Is it github or the indent is really messed up ?

Reply:
It's github, they use 8 spaces tabs.

Check with `?ts=4`